### DISCUSSION TITLE
Question about making views, models and controllers directly from a create_example_table.php

### DISCUSSION THREAD

**Discussion by goatrikv18** [Laravel-Backpack/community-forum#534] [Q&A (Help)] [answered]
Does DevTools offer where I can easily make models, views, controllers directly from my database migrations content?

**Comment by promatik** [comment, marked as answer]
Hi @goatrikv18, that's on our plans 👌

For now DevTools will help you generate EVERYTHING, with one form submit you generate migrations, models, seeders, factories, controller.
We have a POC where we're able to generate all of this based on existing migrations, it will probably be release very soon, stay tunned for updates 🙌
